scripts/zones/Southern_San_dOria/npcs/Kipopo.lua

```diff
@@ -2,38 +2,78 @@
 -- Area: Southern San d'Oria
 --  NPC: Kipopo
 -- Type: Leathercraft Synthesis Image Support
--- !pos -191.050 -2.15 12.285 230
+-- !pos  -191.050 -2.15 12.285 230
 -----------------------------------
-require("scripts/globals/status");
-require("scripts/globals/crafting");
-local ID = require("scripts/zones/Southern_San_dOria/IDs");
+require("scripts/globals/status")
+require("scripts/globals/crafting")
+require("scripts/globals/quests")
+require("scripts/globals/npc_util")
+local ID = require("scripts/zones/Southern_San_dOria/IDs")
 -----------------------------------
 
 function onTrade(player,npc,trade)
+    local sayItWithAHandbagCS = player:getVar("sayItWithAHandbagCS")
```

Comment:
Just noticing. No real reason to make a local unless you are using it more than 1 time per function. Could just make it part of the if statement